src/args.rs

```diff
@@ -44,13 +44,18 @@ pub fn parse() -> ArgMatches<'static> {
                 .arg(
                     Arg::with_name("ignore-nothing")
                         .long("ignore-nothing")
-                        .help("Ignore nothing, not even target/ and .git/"),
+                        .help("Ignore nothing, not even target/, .git/, and .ignore"),
```

Comment:
That's confusing, as `.ignore` is an ignore file, and `target/` and `.git/` are ignored paths. Recommend leaving as is instead of this change

Reply:
Ah yes. Maybe I can word it as “and patterns mentioned in .gitignore or .ignore files”?